components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/core/CarbonAxisConfigurator.java

```diff
@@ -238,6 +241,26 @@ public boolean isGlobalyEngaged(AxisModule axisModule) {
         return false;
     }
 
+    /**
+     * Set the hotDeployment variable after reading the hot deployment parameter from the axis configuration. If the
+     * value is null, then the hotDeployment variable will be set to false which is the default value.
+     */
+    private void setHotDeploymentFeature() {
```

Comment:
Shall we change this method name to updateHotDeploymentParameter? WDYT?

Reply:
Updated.